src/strategies/lrc-l2-nft-balance-of/index.ts

```diff
@@ -0,0 +1,80 @@
+import { subgraphRequest } from '../../utils';
+
+export const author = 'karamorf';
+export const version = '0.1.0';
+
+const LIMIT = 1000;
+
+function makeQuery(snapshot, minter, lastUpdatedAt, blacklisted_accounts) {
+  const query: any = {
+    accountNFTSlots: {
+      __args: {
+        where: {
+          nft_: { minter: minter },
+          account_not_in: blacklisted_accounts,
+          lastUpdatedAt_gt: lastUpdatedAt
+        },
+        first: LIMIT
+      },
+      account: { address: true },
+      balance: true,
+      lastUpdatedAt: true
+    }
+  };
+
+  if (snapshot !== 'latest') {
+    query.accountNFTSlots.__args.where = {
+      ...query.accountNFTSlots.__args.where,
+      _change_block: {
+        number_gte: snapshot
```

Comment:
why `gte`?? is there a reason? ideally, voting power should be based on the snapshot. else there can be inconsistent voting power during the proposal voting period

Reply:
oh, is that what this parameter is doing? we want values before block X for when the proposal was submitted?

Had the roughest of times trying to figure out what this parameter meant. It should probably be `lt` right?

Reply:
@ChaituVR if I do something like this:
```suggestion
        lastUpdatedAtTransaction_lt: snapshot 
```

then it looks like it'd exclude accounts that received tokens after the snapshot. However you wouldn't get the balance at the time of the snapshot, just the current balance. I guess that might be good enough. Is this true of the `lrc-l2-subgraph-balance-of` strategy you wrote? It looks like given my observation here that it'd also apply to in that strategy.

To see this, using this graph query:
```
{
 accountNFTSlots(first: 5, where: { nft_: { minter: "57727"}, lastUpdatedAtTransaction_lt: "28307" }) {
   account { address } 
   balance
   lastUpdatedAtTransaction {
     id
   }
 }
}
```
I'd expect to see a balance of 100k because I didn't move those other tokens out of this account until block `28578`.

Reply:
ahh, I see what's happening in your strategy now. It's in the root __args filter, not within the where filter.